src/FunderMaps.Infrastructure/BatchClient/BatchClient.cs

```diff
@@ -0,0 +1,55 @@
+using System.Text.Json;
+using FunderMaps.BatchNode;
+using FunderMaps.Core.Interfaces;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace FunderMaps.Infrastructure.BatchClient
+{
+    /// <summary>
+    ///     Client connector to the batch node.
+    /// </summary>
+    internal class BatchClient : Batch.BatchClient, IBatchService // TODO: Inherit from AppServiceBase
```

Comment:
Do not inherit from BatchClient

```suggestion
    internal class BatchClient : IBatchService // TODO: Inherit from AppServiceBase
```